plugins/modules/aci_l4l7_device_selection_if_context.py

```diff
@@ -0,0 +1,307 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: aci_l4l7_device_selection_if_context
```

Comment:
My concern with that is we need the ability to delete the child objects, which I am not sure we can do (at least, not without deleting the parent object and re-creating without the child object) if we have them just as child_configs in a parent module instead of as separate modules in their own right.

Reply:
updated module to no longer require child modules